common/test/acceptance/pages/lms/discussion.py

```diff
@@ -381,12 +380,12 @@ def check_threads_rendered_successfully(self, thread_count):
         """
         return len(self.q(css=".forum-nav-thread").results) == thread_count
 
-    def check_window_is_on_top(self):
+    def check_focus_changes(self, selector):
```

Comment:
same here. the method name should be more explicit: like, `check_focus_not_set` or something like that.